core/src/main/res/raw/ayp_youtube_player.html

```diff
@@ -96,6 +99,15 @@
                 YouTubePlayerBridge.sendVideoDuration(videoDuration);
             }
 
+            function sendCurrentVideoIdFromPlaylist(player) {
+                var playlist = player.getPlaylist();
+                if ( typeof playlist !== 'undefined' && Array.isArray(playlist) && playlist.length > 0 ) {
+                    var index = player.getPlaylistIndex();
```

Comment:
are we 100% sure this is always going to be an integer? should we check if it is? can you please send a link to the doc?